### PR TITLE
Add an architecture decision register to the repo

### PR DESCRIPTION
**Pull request by jasonpaige**
### Change description ###
We should be recording the key architectural decisions being made as we
build our services.

I've used the format described by Michael Nygard here: http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: doc/arch/adr-001-only-128-bit-encryption.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# ADR 001: Only use 128 bit encrpytion key length
```
Comment (rkondratowicz) [inline doc/arch/adr-001-only-128-bit-encryption.md]: typo in 'encryption'

File: doc/arch/adr-001-only-128-bit-encryption.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+This was based on research conducted in 2009, however, thinking around this is hasn't changed drastically since.
+
+## Decision
+We will implement 128 bit length keys using AES encryption and monitor the latest thinking on AES encryption to potentially update our approach in the future.
+
+## Status 
+Accepted
+
+## Consequences
+Using only 128 bit length keys sounds less than ideal to people at face value so it's important to reference this decision to educate them why we are not using 256. We must also keep up to date with advances in encrpytion technology to ensure our citizens data remains secure.
```
Comment (rkondratowicz) [inline doc/arch/adr-001-only-128-bit-encryption.md]: same typo here

**Comment by timja-kainos** [comment]
wiki can be edited by anyone at anytime and it can't be reviewed unfortunately :(
you can clone the wiki but its impossible to do a PR to it for review of the actual docs

**Comment by jasonpaige** [comment]
fair point on the CI. We could hack something together to get Travis to skip it as a few have done here https://github.com/travis-ci/travis-ci/issues/6301 but I'd sooner wait until there's something "proper" in place?

**Comment by rkondratowicz** [comment]
@jasonpaige merged with typos?

**Pull request merged**